src/oscar/apps/catalogue/expressions.py

```diff
@@ -32,3 +32,4 @@ class ExpandDownwardsCategoryQueryset(Subquery):
 
     def as_sqlite(self, compiler, connection):
         return super().as_sql(compiler, connection, self.template[1:-1])
+    
```

Comment:
Still a change here, please remove this change.

Reply:
Sorry, just changed it. No changes after line 19.